uk_election_timetables/elections/city_of_london_local.py

```diff
@@ -0,0 +1,40 @@
+from datetime import date
+
+from uk_election_timetables.calendars import working_days_before, Country
+from uk_election_timetables.election import Election
+
+
+"""
+TODO:
+This is insufficently nuanced
+https://github.com/DemocracyClub/uk-election-timetables/issues/8
+"""
+
+
+class CityOfLondonLocalElection(Election):
+    def __init__(self, poll_date: date):
+        """
+        :param poll_date: a datetime representing the date of the poll
+        """
+        Election.__init__(self, poll_date, Country.ENGLAND)
+
+    @property
+    def sopn_publish_date(self) -> date:
+        """
+        Calculate the SOPN publish date for a City of London local election.
+
+        :return: a datetime representing the expected publish date
+        """
+
+        return working_days_before(self.poll_date, 17, super()._calendar())
+
+    @property
+    def registration_deadline(self) -> date:
+        """
+        Calculates the voter registration deadline for a City of London local election.
+
+        :return: a datetime representing the voter registration deadline
+        """
+        if self.poll_date <= date(self.poll_date.year, 2, 15):
+            return date(self.poll_date.year - 2, 11, 30)
+        return date(self.poll_date.year - 1, 11, 30)
```

Comment:
My source for this is basically "@pmk01 told me on slack" :see_no_evil: . The statement "The new register _comes into force_ on 16 February" is non-contraversial. We think that means if there was a City of London by-election with polling day on 16 February that's the first day the new register is used (as opposed to say the Notice of Election being published on 16 February, for example). In any case, this is way closer to being right than what we're saying now which is miles off.